src/main/java/uk/gov/ons/ctp/response/action/representation/ActionRulePutRequestDTO.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.ctp.response.action.representation;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
+/**
+ * Domain model object for representation.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+public class ActionRulePutRequestDTO {
+
+  @Size(max = 100)
+  private String name;
```

Comment:
Why can't we update actionplanname?

Reply:
It's a different resource. This is only for updating action rule details.